lms/templates/courseware/courses.html

```diff
@@ -12,6 +12,13 @@
       <%static:include path="discovery/${template_name}.underscore" />
   </script>
   % endfor
+  <script type="text/javascript">;(function (define) {{
+      define('course_discovery_meanings', function() {{
+          'use strict';
+          return ${course_discovery_meanings};
```

Comment:
Never mind. I see that there are no user provided strings in the course_discovery_meanings.